stock_inventory_barcode/wizard/stock_inventory_barcode.py

```diff
@@ -15,59 +15,70 @@ class StockInventoryBarcode(models.TransientModel):
 
     inventory_id = fields.Many2one(
         'stock.inventory', string='Inventory', required=True)
-    inventory_location_id = fields.Many2one(
-        'stock.location', required=True, string='Root Inventory Location')
+    company_id = fields.Many2one(
+        'res.company', string='Company', required=True)
```

Comment:
Shouldn't this have a default as required ?

Reply:
The default value for company_id is set by the default_get method.